controllers/apply-next/lib/validate-model.js

```diff
@@ -56,6 +56,8 @@ module.exports = function validateModel(formModel) {
     const sectionSchema = Joi.object({
         slug: Joi.string().required(),
         title: Joi.string().required(),
+        shortTitle: Joi.string().optional(),
```

Comment:
This allows sections to have a short title, used in meta title and on the dashboard screen.